resolwe_bio/processes/variant_calling/gatk_select_variants.py

```diff
@@ -0,0 +1,158 @@
+"""Run GATK SelectVariants."""
+from plumbum import TEE
+
+from resolwe.process import (
+    BooleanField,
+    Cmd,
+    DataField,
+    FileField,
+    GroupField,
+    IntegerField,
+    ListField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class GatkSelectVariants(Process):
+    """Select a subset of variants based on various criteria using GATK SelectVariants."""
+
+    slug = "gatk-select-variants"
+    name = "GATK SelectVariants"
+    category = "GATK"
+    process_type = "data:variants:vcf:selectvariants"
+    version = "1.0.0"
+    scheduling_class = SchedulingClass.BATCH
+    entity = {
```

Comment:
Usually the input to this would be a multisample VCF. I think it is best to remove the `entity`

Reply:
You are right most of the time this would not be useful.